rdiodj/static/js/partylist.js

```diff
@@ -87,7 +90,9 @@ app.RoomView = Backbone.View.extend({
           }
         });
       } else {
-        data.nowPlaying = "silence";
+        data.nowPlaying = "sadness";
```

Comment:
Feel free to change it back. I thought it was sad that a room existed without playing music.

> On May 13, 2015, at 3:15 AM, Emily Stumpf <notifications@github.com> wrote:
> 
> In rdiodj/static/js/partylist.js:
> 
> > @@ -87,7 +90,9 @@ app.RoomView = Backbone.View.extend({
> >            }
> >          });
> >        } else {
> > -        data.nowPlaying = "silence";
> > +        data.nowPlaying = "sadness";
> I find "sadness" here a little bit confusing. What's wrong with "silence"?
> 
> —
> Reply to this email directly or view it on GitHub.
>